components/Dialogs/examples/DialogsThemedPresentationExampleViewController.swift

```diff
@@ -0,0 +1,196 @@
+                         relatedBy: .lessThanOrEqual,
+                         toItem: self.view,
+                         attribute: .bottom,
+                         multiplier: 1.0,
+                         constant: 0.0),
+      NSLayoutConstraint(item: confirmButton,
+                         attribute: .centerX,
+                         relatedBy: .equal,
+                         toItem: self.view,
+                         attribute: .centerX,
+                         multiplier: 1.0,
+                         constant: 0.0),
+      ])
+  }
+
+  override var preferredContentSize: CGSize {
+    get {
+      return CGSize(width:200.0, height:150.0);
+    }
+    set {
+      super.preferredContentSize = newValue
+    }
+  }
+
+  @objc func dismissDialog() {
+    self.dismiss(animated:true)
+  }
+}
+
+class DialogsThemedPresentationExampleViewController: UIViewController {
+
+  private let materialButton = MDCButton()
+  private let transitionController = MDCDialogTransitionController()
+  var containerScheme = MDCContainerScheme()
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    view.backgroundColor = containerScheme.colorScheme?.backgroundColor ??
+      MDCSemanticColorScheme(defaults: .material201804).backgroundColor
+
+    materialButton.translatesAutoresizingMaskIntoConstraints = false
+    materialButton.setTitle("Custom Alert With Themed Presentation", for: .normal)
+    materialButton.sizeToFit()
+    materialButton.addTarget(self, action: #selector(tap), for: .touchUpInside)
+    materialButton.applyTextTheme(withScheme: containerScheme)
+    self.view.addSubview(materialButton)
+
+    NSLayoutConstraint.activate([
+      NSLayoutConstraint(item:materialButton,
+                         attribute: .centerX,
+                         relatedBy: .equal,
+                         toItem: self.view,
+                         attribute: .centerX,
+                         multiplier: 1.0,
+                         constant: 0.0),
+      NSLayoutConstraint(item: materialButton,
+                         attribute: .centerY,
+                         relatedBy: .equal,
+                         toItem: self.view,
+                         attribute: .centerY,
+                         multiplier: 1.0,
+                         constant: 0.0)
+      ])
+  }
+
+  @objc func tap(_ sender: Any) {
+    let title = "Reset Settings"
+    let message = "Your device will reset to its default factory settings."
+    let customDialogController = CustomDialogViewController(title: title, message: message)
+    customDialogController.modalPresentationStyle = .custom;
+    customDialogController.transitioningDelegate = self.transitionController;
+
+    if let presentationController = customDialogController.mdc_dialogPresentationController {
+      presentationController.applyTheme(withScheme: containerScheme)
+      customDialogController.view.layer.cornerRadius = presentationController.dialogCornerRadius;
```

Comment:
I don't understand - isn't that literally what `dialogCornerRadius` is supposed to do? Isn't it supposed to set the dialog's corner radius?

@galiak11 may be able to help us here.  Seems like the API is confusing if it's not actually setting corner radius values.

Reply:
dialogCornerRadius only adjusts the corner radius of the shadow view contained by the presentation container per my understanding on the code implementation.

@galiak11 not sure if it was intentional or a bug.

Reply:
@wenyuzhang666 Per our discussion - please open a bug for this, and add it as a todo comment here.

Reply:
Addressed in 77d9e69272b8225561bc427c481ee5505d386a41